RyleyGG/apluschiever
# Dashboard Development + Lesson Page

# Issue
#35 


# Description
Dashboard was developed to allow users to view all available courses, add courses to enrolled, and remove courses from enrolled.
On the dashboard, courses were connected to the course view page to allow users to navigate to the course graph.
Graph nodes were connected to a lesson page skeleton, which currently only shows the name of the lesson and a basic setup.

# Type Of Change

# Test Steps
Regular tests of nuking and rerunning application, checking to see that endpoints were set up correctly in fastapi and database works correctly by testing adding and removing courses. 

# Checklist

## Formatting
Please use markdown in your pull request message. A useful summary of commands can be found [here](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf).
